solidity/contracts/bridge/library/Fraud.sol

```diff
@@ -0,0 +1,396 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
+import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
+import {CheckBitcoinSigs} from "@keep-network/bitcoin-spv-sol/contracts/CheckBitcoinSigs.sol";
+import "../../GovernanceUtils.sol";
+import "./EcdsaLib.sol";
+import "../Bridge.sol";
+
+library Fraud {
+    using BytesLib for bytes;
+    using BTCUtils for bytes;
+    using BTCUtils for uint32;
+    using EcdsaLib for bytes;
+
+    struct FraudChallenge {
+        // The address of the party challenging the wallet.
+        address challenger;
+        // The amount of ETH the challenger deposited.
+        uint256 depositAmount;
+        // The timestamp the challenge was submitted at.
+        uint32 reportedAt;
+        // The flag indicating whether the challenge has been closed.
+        bool closed;
+    }
+
+    struct Data {
```

Comment:
Can we put the `Data` struct on top of the library? We typically do it that way in other repos so it may help with navigation.

Reply:
Done in 1dd89b1b03e42f986817f43eb8c9c7b85b4f77ee.